spec/v2/index.bs

```diff
@@ -1629,10 +1638,28 @@ With action-specific response parameters
 
 ## <dfn>Action ListFootprints</dfn> ## {#api-action-list}
 
-Lists [[#dt-pf|product footprints]] with OPTIONAL filtering by properties <{ProductFootprint/created}> or <{ProductFootprint/updated}>. 
+Lists [[#dt-pf|product footprints]] with [[#api-action-list-pagination|pagination]] and optional [[#api-action-list-filtering|filtering]].
 
 [=Host systems=] SHOULD implement an access management system and only return the [[#dt-pf|product footprints]] for which the [=data owner=] granted access to the requesting [=data recipient=].
 
+### Filtering ### {#api-action-list-filtering}
```

Comment:
I have no context or knowledge regarding filtering and/or why this is changing, so I can't provide a proper review. Just letting you know that I can't actually provide a proper review and defer to others who can.